src/shared/assets/svg/Basket.tsx

```diff
@@ -1,6 +1,25 @@
+  },
+  lg: {
+    width: 36,
+    height: 32,
+    pathDefinition: "M34.0527 22.418H10.791L11.959 20.0391L31.3652 20.0039C32.0215 20.0039 32.584 19.5352 32.7012 18.8867L35.3887 3.84375C35.459 3.44922 35.3535 3.04297 35.0957 2.73438C34.9683 2.5825 34.8093 2.46016 34.6299 2.37585C34.4505 2.29153 34.2549 2.24726 34.0566 2.24609L9.36914 2.16406L9.1582 1.17188C9.02539 0.539062 8.45508 0.078125 7.80664 0.078125H1.77148C1.40578 0.078125 1.05505 0.223402 0.79645 0.481997C0.537855 0.740592 0.392578 1.09132 0.392578 1.45703C0.392578 1.82274 0.537855 2.17347 0.79645 2.43207C1.05505 2.69066 1.40578 2.83594 1.77148 2.83594H6.68945L7.61133 7.21875L9.88086 18.207L6.95898 22.9766C6.80725 23.1814 6.71585 23.4245 6.69514 23.6786C6.67443 23.9326 6.72523 24.1874 6.8418 24.4141C7.07617 24.8789 7.54883 25.1719 8.07226 25.1719H10.5254C10.0024 25.8665 9.71994 26.7126 9.7207 27.582C9.7207 29.793 11.5176 31.5898 13.7285 31.5898C15.9395 31.5898 17.7363 29.793 17.7363 27.582C17.7363 26.7109 17.4473 25.8633 16.9316 25.1719H23.2246C22.7016 25.8665 22.4192 26.7126 22.4199 27.582C22.4199 29.793 24.2168 31.5898 26.4277 31.5898C28.6387 31.5898 30.4355 29.793 30.4355 27.582C30.4355 26.7109 30.1465 25.8633 29.6309 25.1719H34.0566C34.8144 25.1719 35.4355 24.5547 35.4355 23.793C35.4333 23.4276 35.2866 23.078 35.0275 22.8204C34.7685 22.5629 34.4181 22.4182 34.0527 22.418ZM9.94336 4.88281L32.4355 4.95703L30.2324 17.293L12.5645 17.3242L9.94336 4.88281ZM13.7285 28.8164C13.0488 28.8164 12.4941 28.2617 12.4941 27.582C12.4941 26.9023 13.0488 26.3477 13.7285 26.3477C14.4082 26.3477 14.9629 26.9023 14.9629 27.582C14.9629 27.9094 14.8328 28.2234 14.6013 28.4549C14.3699 28.6864 14.0559 28.8164 13.7285 28.8164ZM26.4277 28.8164C25.748 28.8164 25.1934 28.2617 25.1934 27.582C25.1934 26.9023 25.748 26.3477 26.4277 26.3477C27.1074 26.3477 27.6621 26.9023 27.6621 27.582C27.6621 27.9094 27.5321 28.2234 27.3006 28.4549C27.0691 28.6864 26.7551 28.8164 26.4277 28.8164Z",
+  },
+};
+
+export const Basket = ({ className = "fill-[#131313]", type }: BasketIconProp): JSX.Element => {
+  const { width, height, pathDefinition } = basketType[type];
+  return <svg width={width} height={height} viewBox={`0 0 ${width} ${height}`} fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path className={className} d={pathDefinition}/>
+    {/* <circle cx="20" cy="5" r="5" fill="#FF3232"/> it will show that basket has products */}
```

Comment:
In general when u want to add some proposal it is good practice to use "TODO","FIXME","BUG".
Read more: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree